src/components/victory-line.jsx

```diff
@@ -89,6 +90,11 @@ export default class VictoryLine extends React.Component {
      */
     label: React.PropTypes.string,
     /**
+     * The labelComponent prop takes in an entire, HTML-complete label component
+     * which will be used to create labels for line to use
+     */
+    labelComponent: React.PropTypes.element,
+    /**
```

Comment:
yeah, good point. there's not much of a reason to use the built ins. I guess I don't understand why `React.PropTypes.element` is a bad choice then. I get that semantically we're using it more like a class, especially with things like bar and scatter... I'm using `cloneElement` under the covers anyway to keep whatever props the labelComponent comes in with. Also, if we eliminate the `label` prop, shouldn't `labelComponent` take an instance so that it can have a labelText prop or child?